src/ngraph/runtime/cpu/builder/quantized_conv.cpp

```diff
@@ -106,20 +106,61 @@ namespace ngraph
                     auto& out0_tensor = external_function->get_tensor_data(out[0].get_name());
 
                     auto& mkldnn_emitter = external_function->get_mkldnn_emitter();
+                    bool is_bias_i32 = (args[2].get_element_type() == element::i32);
```

Comment:
Cant we define QuantizedConvolutionBias as taking int32 bias and insert a reorder in the builder itself instead of doing it here?

Reply:
tailoring the op to a specific type seems overly optimized to the current graphs we are working on unless I am missing something here.  can we always count on bias being a certain type?